parseigc.js

```diff
@@ -287,24 +287,24 @@ function parseIGC(igcFile) {
         name: 'Date',
         value: displayDate
     });
-
+    //This bit is no longer needed as we won't be plotting these lines anyway.
     // If the file contains a declared task, then it may start or end with
     // 'turnpoints' at 0 degrees N, 0 degrees E. These represent the takeoff
     // and landing positions, which could not have been known at the time when
     // the declaration was made. Replace them with the actual values so that
     // the task can be plotted correctly on a map.
 
-    if (model.task.coordinates.length > 0) {
-        if (model.task.coordinates[0][0] === 0.0 && model.task.coordinates[0][1] === 0.0) {
-            model.task.coordinates[0] = model.latLong[0];
-            model.task.names[0] = 'Takeoff';
-        }
-        var lastTask = model.task.coordinates.length - 1;
-        if (model.task.coordinates[lastTask][0] === 0.0 && model.task.coordinates[lastTask][1] === 0.0) {
-            model.task.coordinates[lastTask] = model.latLong[model.latLong.length - 1];
-            model.task.names[lastTask] = 'Landing';
-        }
-    }
+   // if (model.task.coordinates.length > 0) {
+       // if (model.task.coordinates[0][0] === 0.0 && model.task.coordinates[0][1] === 0.0) {
+            //model.task.coordinates[0] = model.latLong[0];
+          //  model.task.names[0] = 'Takeoff';
+        //}
+     //   var lastTask = model.task.coordinates.length - 1;
+       // if (model.task.coordinates[lastTask][0] === 0.0 && model.task.coordinates[lastTask][1] === 0.0) {
+            //model.task.coordinates[lastTask] = model.latLong[model.latLong.length - 1];
+           // model.task.names[lastTask] = 'Landing';
+        //}
+   // }
```

Comment:
I generally prefer to delete code that is no longer needed instead of commenting it out. The old code can still be recovered by looking back through the revision history in Git.